includes/express-checkout/class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -123,7 +123,7 @@ public function display_express_checkout_buttons() {
 		$should_show_woopay                  = $this->platform_checkout_button_handler->should_show_woopay_button();
 		$should_show_express_checkout_button = $this->express_checkout_helper->should_show_express_checkout_button();
 
-		// When Payment Request button is enabled, we need the separator markup on the page, but hidden in case the browser doesn't have any payment request methods to display.
+		// When Express Checkout button is enabled, we need the separator markup on the page, but hidden in case the browser doesn't have any express methods to display.
```

Comment:
```suggestion
		// When Express Checkout button is enabled, we need the separator markup on the page, but hidden in case the browser doesn't have any express payment methods to display.
```
Mm? Not really a critical suggestion. 🤷